### PR TITLE
Healthcheck auto assigner of doom

### PR DESCRIPTION
**Pull request by andrewconner**

### CODE REVIEW

**Comment by andrewconner** [comment]
@eishay  :)

**Comment by gmethvin** [comment]
I appreciate your fixing the spelling of Léo's name.

File: kifi-backend/modules/common/app/com/keepit/common/healthcheck/Healthcheck.scala [hunk 1]
```diff
@@ -38,7 +38,30 @@ object Healthcheck {
...
+      33 -> STEPHEN,
+      34 -> ANDREW, // 2013/08/19
+      35 -> GREG,
+      36 -> LÉO,
+      37 -> EISHAY,
+      38 -> STEPHEN,
+      39 -> ANDREW,
+      40 -> GREG,
+      41 -> LÉO,
+      42 -> EISHAY,
```
Comment (eishay) [inline kifi-backend/modules/common/app/com/keepit/common/healthcheck/Healthcheck.scala:55]: Oh, if we're into spelling our names the right way, then my name is actually `ישי`. Its much shorter and now you can pronounce it right :-)

**Pull request merged**